### DISCUSSION TITLE
Глоссарий (словарь терминов и переводов) для русскоязычной документации

### DISCUSSION THREAD

**Discussion by dsvitnev** [vuejs-translations/docs-ru#55] [Show and tell]
Я обнаружил нехватку глоссария для обеспечения консистентности переводов между контрибьюторами ру доки, поэтому я [сделал его](https://gist.github.com/dsvitnev/92d985fbe1dda36e898618e59813d60e) 😄.

Считаю наличие глоссария необходимым, во избежание чувства расстеряности у контрибьюторов и создания многочисленных вопросов и обсуждения по типу #51 

Предлагаю открыть вики в проекте и вставить его туда как один из разделов, а после можно и обсудить отдельные термины и их варианты перевода.

@Alex-Sokolov, что скажешь?

**Comment by Alex-Sokolov** [comment]
@dsvitnev к некоторым переводам я уже иным привык конечно, но в принципе я не против) можно в README добавить например ссылку на него или даже отдельным файлом тут в репо сделать

**Comment by dsvitnev** [comment]
@Alex-Sokolov здорово) активно ворвусь после 15 числа; нужно будет провести ревизию переведенного корпуса и приведение в соответствие с глоссарием, и составить орфографические гайдлайны — уже бросилось в глаза, что написание некоторых лексем и оборотов идёт в разрез с правилами

По поводу места размещения — всё же предлагаю создать wiki в репозитории, сделать на него ссылку в README.md и поместить туда глоссарий, это [просто](https://docs.github.com/en/communities/documenting-your-project-with-wikis/about-wikis)
Потому что:
1.  Линк на мой гист с глоссарием в readme — слишком много чести мне 😄 , я могу случайно удалить гист, и самое главное, кто-то другой не сможет его актуализировать в случае необходимости
2. Отдельный файл в репе — в целом вариант, но он не обладает преимуществом размещения как раздела в вики (ниже)
3. В вики можно удобно навигироваться по якорям-буквам на странице с глоссарием, см. [пример](https://github.com/vuejs-translations/docs-uk/wiki/%D0%93%D0%BB%D0%BE%D1%81%D0%B0%D1%80%D1%96%D0%B9), что удобнее чем скролл или cmd+f (по крайней мере, для кого-то). Плюс вики можно будет наполнить гайдлайнами и справочной информацией для начала перевода, плюс это конвенциональный способ аггрегации справочной информации по переводу vue-документации, см. примеры [раз](https://github.com/vuejs-translations/docs-uk/wiki), [два](https://github.com/vuejs-translations/docs-fr/wiki), [три](https://github.com/vuejs-translations/docs-zh-cn/wiki), [четыре](https://github.com/vuejs-translations/docs-ja/wiki)

**Reply by Alex-Sokolov** [reply]
Супер, можно и в Wiki создать, главное чтобы другим удобно тоже было

**Reply by farid-temuri** [reply]
@Alex-Sokolov @dsvitnev Я зашел с предложением создать вики с глоссарием, но тут оказались вы. Думаю уже время это сделать, так как новые переводы уже требуются, но глоссария нету и свежие переводы могут быть неправильными для кого-то, но при это и правил тоже не будет, что может приводить в замешательство и сильно замедлить процесс перевода

**Reply by farid-temuri** [reply]
Вики на самом деле легко создать и ,насколько я знаю, уже права нужные есть как у @Alex-Sokolov @Ibochkarev

**Reply by Alex-Sokolov** [reply]
Мне казалось у всех права для wiki есть. Вот первую страницу создал, есть возможность правки в неё вносить?

**Reply by farid-temuri** [reply]
@Alex-Sokolov не могу редактировать. Мб через форк смогу. Позже протестирую еще.

**Reply by Ibochkarev** [reply]
@farid-temuri можно проверить. Внес изменения в настройках

**Reply by farid-temuri** [reply]
@Ibochkarev Могу 👍
